server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -98,6 +99,16 @@ const accountMutations = {
           }
         }
 
+        models.Activity.create({
+          type: activities.COLLECTIVE_EDITED,
+          UserId: req.remoteUser.id,
+          UserTokenId: req.userToken?.id,
+          CollectiveId: account.id,
+          FromCollectiveId: account.id,
+          HostCollectiveId: account.approvedAt ? account.HostCollectiveId : null,
+          data: { previousData: { [args.key]: account.data?.[args.key] }, newData: { [args.key]: args.value } },
```

Comment:
`account.data` - are we not editing `settings` here? Also, I would make sure to preserve the existing structure:

```suggestion
          data: { previousData: { settings: previousSettings }, newData: { settings: newSettings } },
```